src/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Leaderboard</title>
+  </head>
+  <body>
+    <header>
+      <h1>Leaderboard</h1>
+    </header>
```

Comment:
Well done nailing the wireframe layout! 💯 

![image](https://github.com/shihabrafio/Leaderboard/assets/38088818/30eb96bc-6eb6-4d09-89d3-8a532bf78181)